src/components/AssetItemNew/AssetItem.js

```diff
@@ -0,0 +1,133 @@
+// @flow
+
+import React from 'react'
+import classNames from 'classnames'
+import { connect } from 'react-redux'
+
+import {
+  JAssetSymbol,
+  JIcon,
+  JLink,
+  JShimmer,
+} from 'components/base'
+import { selectDigitalAssetsItems } from 'store/selectors/digitalAssets'
+import { selectBalanceByAssetAddressToCurrentBlock } from 'store/selectors/balances'
+import { selectSettingsFiatCurrencyData } from 'store/selectors/settings'
+import { getFiatBalance } from 'store/utils/getFiatBalances'
+import {
+  divDecimals,
+  formatBalance,
+} from 'utils/numbers'
+import { checkBalanceLoading } from 'utils/digitalAssets'
+import { formatAssetBalance } from 'utils/formatters'
+import { type ToBigNumberValue } from 'utils/numbers/toBigNumber'
+
+import offsetsStyle from 'styles/offsets.m.scss'
+import assetItemStyle from './assetItem.m.scss'
+
+type ContainerProps = {|
+  +address: AssetAddress,
+|}
+
+type Props = ContainerProps
+  & DigitalAssetWithBalance
+  & {
+  balance: ToBigNumberValue,
+  fiatSymbol: string,
+  fiatBalance: string,
+  isLoadingBalance: boolean,
+}
+
+function AssetItem(props: Props) {
+  // const { asset } = props
```

Comment:
Don't forget to remove it

Reply:
Sure, but I still want to handle unexpected cases